extensions/servlet/test/com/google/inject/servlet/ServletDefinitionPathsTest.java

```diff
@@ -93,6 +93,12 @@ protected void service(HttpServletRequest servletRequest,
     expect(request.getServletPath())
         .andReturn(requestPath);
 
+    // computePath()
```

Comment:
ah sorry missed that it's new.  i'd use a comment with english if you want to explain that this is similar to something else, rather than commented out code that looks like it's stale.